basis/ui/tools/tools.factor

```diff
@@ -34,6 +33,28 @@ tool "common" f {
     { T{ key-down f f "F11" } toggle-fullscreen }
 } ? prepend define-command-map
 
+
+tool "tool-switching" f {
```

Comment:
Is this section duplicative of the above code?

Reply:
Not even sure how I wrangled this - sorry for making your job difficult, will be more careful next time. Should be fixed on the next commit.

Reply:
I don't mind difficult!

It's nice to get contributions and new features!